libc/spec/stdc_ext.td

```diff
@@ -7,6 +7,13 @@ def StdcExt : StandardSpec<"stdc_ext"> {
       [],  // types
       [],  // enums
       [    // functions
+          GuardedFunctionSpec<"abshr", RetValSpec<ShortFractType>, [ArgSpec<ShortFractType>], "LIBC_COMPILER_HAS_FIXED_POINT">,
```

Comment:
given that these are in stdfix.h, would it be reasonable to assume that the compiler has fixed point support?

Reply:
I think we should still put it in the guards, since the types still require specific compiler flag, and if the `stdfix.h` is included somewhere transitively in the users' codes where it's not being used, and they cannot easily control the header inclusions, it will force the users to add the compiler flag, or a hard compile errors.

By keeping the guards here, users can explicitly control when to enable it with the compiler flags.

Reply:
in that case can we add a `#warning` to stdfix.h for if they don't have fixed point available?

Reply:
I don't think we need to add that, because right now, if users try to use fixed point types / functions / macros without the compile flag, it will be a hard compile error.  And if we add the warnings, users will keep seeing them even if they don't use it and got it transitively.  And in a worse scenario, where they try to build with `-Werror`, those warnings will become hard errors.